internal/lib/control_config.go

```diff
@@ -121,7 +131,22 @@ func (c *akoControlConfig) SetAKOInstanceFlag(flag bool) {
 func (c *akoControlConfig) GetAKOInstanceFlag() bool {
 	return c.primaryaAKO
 }
-
+func (c *akoControlConfig) SetAKOBlackListedNamespaces(nsList []string) {
+	sort.Strings(nsList)
+	val := strings.Join(nsList, ":")
+	cksum := utils.Hash(val)
+	if c.blockedNS.nsChecksum != cksum {
```

Comment:
Are we planning to listen to configmap updates for this?

Reply:
For current release no. For AKO, it will become overwhelming to handle bulk updates of all objects when namespaces transitioned from blocked state to non-blocked state.